react/features/prejoin/components/dialogs/JoinByPhoneDialog.js

```diff
@@ -0,0 +1,245 @@
+// @flow
+
+import React, { Component } from 'react';
+import {
+    dialOut as dialOutAction,
+    joinConferenceWithoutAudio as joinConferenceWithoutAudioAction,
+    openDialInPage as openDialInPageAction
+} from '../../actions';
```

Comment:
Please sort the imports (for other files too) and separate the blocks with empty lines. The preferred order is:
- packages
- own modules in depth order and alphabetically (e.g. `../../something` comes before `../anything`)

See example here: https://github.com/jitsi/jitsi-meet/blob/eed5e31ec9c78d6bc317c27412036e73a974ac9e/react/features/chat/components/native/Chat.js#L3

Reply:
Done.